docs/source/topics/con_differences-from-production-openshift-install.adoc

```diff
@@ -10,3 +10,11 @@
 ** For the same reason, there is no upgrade path to newer OpenShift versions.
 * The OpenShift instance runs in a virtual machine.
 This may cause other differences, particularly with external networking.
+
+{prod} also includes the following non-customizable cluster settings.
+These settings should not be modified:
+
+* Use of the ***.crc.testing** domain.
```

Comment:
@robin-owen this might be changeable, we need to look bit more closely apart from this all other points are valid.

Reply:
@praveenkumar Should we remove this as a non-customizable setting in that case?

Reply:
@robin-owen I think let's have it here atm and then when we have a solution we can actually try to remove it.